src/frontend/app/store/actions/organisation.actions.ts

```diff
@@ -29,6 +42,52 @@ export class GetOrganisation extends CFStartAction implements ICFAction {
   options: RequestOptions;
 }
 
+export class GetAllOrganisationSpaces extends CFStartAction implements PaginatedAction, EntityInlineChildAction {
+  constructor(public paginationKey: string, public orgGuid: string, public cnsi: string) {
+    super();
+    this.options = new RequestOptions();
+    this.options.url = `organizations/${orgGuid}/spaces`;
+    this.options.method = 'get';
+    this.parentGuid = orgGuid;
+  }
+  actions = [GET_ORGANISATION_SPACES, GET_ORGANISATION_SPACES_SUCCESS, GET_ORGANISATION_SPACES_FAILED];
+  entity = SpacesSchema;
+  entityKey = spaceSchemaKey;
+  options: RequestOptions;
+  flattenPagination = true;
+  initialParams = {
+    'results-per-page': 100,
+    'inline-relations-depth': '1'
+  };
+  parentGuid: string;
+}
+
+export const OrganisationWithSpaceSchema = new EntityInlineParent([
+  {
+    path: 'entity.spaces',
+    createAction: (organisation) => {
```

Comment:
It would be nice if this all happened as transparently as possible. Could we give the EntityInlineParent enough information to generate actions from the schema? If we did this we could then remove the path and create action method. Lets chat about it.